conf/services/capital-gains-tax-uk-property.cy.yml

```diff
@@ -0,0 +1,16 @@
+serviceName: Treth Enillion Cyfalaf
+serviceHeaderName: Treth Enillion Cyfalaf
+serviceDescription: Gallwch ddefnyddio’r gwasanaeth hwn i roi gwybod am warediad o dir neu eiddo preswyl yn y DU (a wnaed o 6 Ebrill 2020 ymlaen). Gallwch hefyd fwrw golwg dros eich Ffurflenni Treth blaenorol a’r rhai cyfredol.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /capital-gains-tax-uk-property/start/report-pay-capital-gains-tax-uk-property
```

Comment:
I think the service url should stop at /capital-gains-tax-uk-property

Reply:
Agree as long as no other journey or service which has had an audit and has separate accessibility statement uses same top level URL as these need to be unique.